ChronoStore/CMakeLists.txt

```diff
@@ -1,12 +1,18 @@
-cmake_minimum_required(VERSION 3.19)
+cmake_minimum_required(VERSION 3.25)
 
 find_package(Thallium REQUIRED)
 find_package(HDF5 REQUIRED COMPONENTS C CXX)
 find_package(spdlog REQUIRED)
 
-include_directories(include
-    ../ChronoAPI/ChronoLog/include
-    ../ChronoKeeper)
+message("Chronostore CMakeLists - CMAKE_SOURCE_DIR=${CMAKE_SOURCE_DIR}")
+message("Chronostore CMakeLists - CMAKE_CURRENT_SOURCE_DIR=${CMAKE_CURRENT_SOURCE_DIR}")
+message("Building CMAKE_CURRENT_SOURCE_DIR=${CMAKE_CURRENT_SOURCE_DIR}")
+message("build target : chrono_store")
+
+include_directories(
+    ${CMAKE_CURRENT_SOURCE_DIR}/include
+    ${CMAKE_SOURCE_DIR}/ChronoAPI/ChronoLog/include
+    ${CMAKE_SOURCE_DIR}/ChronoKeeper)
 
 option(BUILD_TEST_DIRECTORY "Build Test Directory" ON)
 if(BUILD_TEST_DIRECTORY)
```

Comment:
For me, I have to add `#include <hdf5.h>` to `ChronoStore/include/StoryReader.h` and `ChronoStore/include/StoryWriter.h` because it cannot understand `hid_t`.  So pls move the inclusion of `hdf5.h` from `ChronoStore/src/StoryReader.cpp` and `ChronoStore/src/StoryWriter.cpp` to the corresponding header files.